### PR TITLE
feat: add simple description to showcase app

### PR DESCRIPTION
**Pull request by kpanot**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

**Review by cpaulve-1A — APPROVED** [review]

File: apps/showcase/src/app/home/home.template.html [hunk 1]
```diff
@@ -1,6 +1,41 @@
 <div class="d-flex flex-column gap-2 align-items-center justify-center flex-fill">
-  <h1>Otter</h1>
-  <p>This is a showcase application.</p>
-  <p>You can find examples of Otter features with some explanations.</p>
-  <img width="500" [src]="'otter-a.svg' | dynamicContent" class="otter-logo">
+  <img width="350" [src]="'otter-a.svg' | dynamicContent" class="otter-logo" alt="Otter">
+  <h1 class="h1 display-1 text-center mt-0 mb-5"><b>Build easily a customizable Angular based application</b></h1>
```
Comment (fpaul-1A) [inline apps/showcase/src/app/home/home.template.html:3]: I feel the header looks way too big with `display-1`

File: apps/showcase/src/app/home/home.template.html [hunk 2]
```diff
@@ -1,6 +1,41 @@
...
+  <h2 class="h4 mt-5 pt-5 text-secondary">
+    The Otter project is a highly modular framework whose goal is to provide a common platform to <b>accelerate</b> and <b>facilitate</b> the development of <span class="text-danger">Angular</span> based Web Applications.<br />
+    The framework offer the capability to customize the application using <b>Content Management System</b>.
```
Comment (fpaul-1A) [inline apps/showcase/src/app/home/home.template.html:6]:
```suggestion
    The framework offers the capability to customize the application using <b>Content Management System</b>.
```

File: apps/showcase/src/app/home/home.template.html (hunk 1 above)
Comment (fpaul-1A) [inline apps/showcase/src/app/home/home.template.html:3]:
```suggestion
  <h1 class="h1 display-1 text-center mt-0 mb-5"><b>Easily build a customizable Angular based application</b></h1>
``` 
?

File: apps/showcase/src/app/home/home.template.html [hunk 3]
```diff
@@ -1,6 +1,41 @@
...
+  </h2>
 </div>
+
+<div class="get-start-section d-flex flex-column gap-2 align-items-center justify-center flex-fill">
+  <h3 class="display-3 text-center mt-0 mb-5">Get started with Otter</h3>
+  <p class="text-secondary">
+    To start a new project you just need to follow the guide <a target="_blank" href="https://github.com/AmadeusITGroup/otter/blob/main/docs/core/START_NEW_APPLICATION.md">on this page</a> or use the following command lines:
```
Comment (fpaul-1A) [inline apps/showcase/src/app/home/home.template.html:13]: the link is not very meaningful out of context
maybe something like
```suggestion
    To start a new project you just need to follow the <a target="_blank" href="https://github.com/AmadeusITGroup/otter/blob/main/docs/core/START_NEW_APPLICATION.md">start new application guide</a> or use the following command lines:
```

Comment (kpanot) [reply apps/showcase/src/app/home/home.template.html:13]: A bit too many "start" in the short sentence

Comment (fpaul-1A) [reply apps/showcase/src/app/home/home.template.html:13]:
```suggestion
    To start a new project you just need to follow the <a target="_blank" href="https://github.com/AmadeusITGroup/otter/blob/main/docs/core/START_NEW_APPLICATION.md">new application guide</a> or use the following command lines:
```
:D

**Review by fpaul-1A — DISMISSED** [review]

File: apps/showcase/src/app/home/home.template.html (hunk 1 above)
Comment (kpanot) [reply apps/showcase/src/app/home/home.template.html]: image
What do you think?

**Review by fpaul-1A — APPROVED** [review]

File: packages/@o3r/components/builders/component-extractor/index.js [hunk 4]
```diff
@@ -8,5 +8,6 @@ It should not be part of the package.
 const {resolve} = require('node:path');
 
 require('ts-node').register({ project: resolve(__dirname, '..', '..', 'tsconfig.builders.json') });
+require('ts-node').register = () => {};
```
Comment (matthieu-crouzet) [inline packages/@o3r/components/builders/component-extractor/index.js:11]: Why do we have to add that ?

**Review by matthieu-crouzet — APPROVED** [review]

File: packages/@o3r/components/builders/component-extractor/index.js (hunk 4 above)
Comment (kpanot) [reply packages/@o3r/components/builders/component-extractor/index.js:11]: Workaround to the multiwatcher builder case that was calling register several times for each child builder.
This is a way (did not really found another :S) to have an unique call to the register function

**Pull request merged**